server/controllers/stock/core.js

```diff
@@ -108,6 +109,7 @@ function getLotFilters(parameters) {
   filters.equals('invoice_uuid', 'invoice_uuid', 'm');
   filters.equals('purchase_uuid', 'origin_uuid', 'l');
   filters.equals('tag_uuid', 'tags', 't');
+  filters.equals('stock_requisition_uuid', 'uuid', 'sr');
```

Comment:
Let's make this a custom filter that looks up the value in requisition movements similar to how the `only_user` works:

```js

  filters.custom(
    'user_id',
    'd.uuid IN (SELECT depot_permission.depot_uuid FROM depot_permission WHERE depot_permission.user_id = ?)',
  );

```

Reply:
here I did not understand very well